website/docs/tutorials/07-testing/02-cheatcodes/mock-call.md

```diff
@@ -4,7 +4,11 @@
 def mock_call(contract_address: int, fn_name: str, ret_data: Union[List[int], Dict]) -> Callable: ...
 ```
 
-Mocks all calls to function with the name `fn_name` of a contract with an address `contract_address` until the returned callable is called. Mocked call returns data provided in `ret_data`. Mock works globally, for all of the contracts, not only the testing contract.
+Mocks all calls to function with the name `fn_name` of a contract with an address `contract_address`, until the returned callable is called. 
+
+Mocked call returns data provided in `ret_data`.
+
+Mock works globally, for all contracts, not only the testing contract.
```

Comment:
I mean this PR doesn't change anything rn. Cannot we just close it? (I am ok if we want split it into separate lines)